.ci/.docker-images.yml

```diff
@@ -277,15 +277,29 @@ images:
   - <<: *apm-agent-nodejs-docker-images
     tag: "8.6"
     build_opts: "--build-arg NODE_VERSION=8.6"
-
-  # APM Agent Ruby Docker images
-
+#
+#  # APM Agent Ruby Docker images
+#
+#  - <<: *apm-agent-ruby-docker-images
+#    name: "apm-agent-jruby"
+#    working_directory: ".ci/docker/jruby"
+#    build_script: "./run.sh --action build --registry ${REGISTRY}/${PREFIX}"
+#    test_script: "./run.sh --action test --registry ${REGISTRY}/${PREFIX}"
+#    push_script: "./run.sh --action push --registry ${REGISTRY}/${PREFIX}"
+
+  # dockerhub
   - <<: *apm-agent-ruby-docker-images
     name: "apm-agent-jruby"
     working_directory: ".ci/docker/jruby"
-    build_script: "./run.sh --action build --registry ${REGISTRY}/${PREFIX}"
-    test_script: "./run.sh --action test --registry ${REGISTRY}/${PREFIX}"
-    push_script: "./run.sh --action push --registry ${REGISTRY}/${PREFIX}"
+    build_script: |
+      EXCLUDE=7-jdk ./run.sh --action build --registry 'elasticobservability'
```

Comment:
I know this is a draft PR, but I wonder whether this automation should not know anything about what versions should be built, but delegate that responsibility to the `run.sh`, what do you think?

Reply:
do you mean the `EXCLUDE` part?

In that case, I kind of agree.. I was torn between the 3 options of 
- using `EXCLUDE` here
- removing the directory entirely
- or excluding in the the `run.sh` script

Reply:
> do you mean the EXCLUDE part?

Yes - sorry for my vague sentence!

Reply:
FYI https://github.com/elastic/apm-agent-ruby/pull/1367/commits/1089887c126a47ca31be9e7d313aa120a6b4c846